docs/intro.rst

```diff
@@ -237,7 +237,7 @@ This is the current list of error and warning codes:
 | E116     | unexpected indentation (comment)                                     |
 +----------+----------------------------------------------------------------------+
 +----------+----------------------------------------------------------------------+
-| E121 (*^)| continuation line under-indented for hanging indent                  |
+| E121(\*^)| continuation line under-indented for hanging indent                  |
```

Comment:
All of the other errors have spaces between the `(` and the error code. Re-aligning the table is quite simple to do if you use vim or sublime text